src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -819,7 +819,15 @@ public static (string owner, string repo, int id) ParsePullRequestUri(string uri
         /// <returns></returns>
         public Task CommitFilesAsync(List<GitFile> filesToCommit, string repoUri, string branch, string commitMessage)
         {
-            return this.CommitFilesAsync(filesToCommit, repoUri, branch, commitMessage, _logger, Client.Credentials.Password);
+            return this.CommitFilesAsync(
+                filesToCommit, 
+                repoUri, 
+                branch, 
+                commitMessage, 
+                _logger, 
+                Client.Credentials.Password, 
+                "dotnet-maestro[bot]", 
```

Comment:
Does adding the [bot] at the end make it so it shows up like this?

![image](https://user-images.githubusercontent.com/23242101/62164911-195e5680-b2d2-11e9-858b-4e1f091f7a7e.png)

or will it start showing up as dotnet-maestro[bot]?
That's fancy if so.